.github/workflows/build.yml

```diff
@@ -79,7 +79,14 @@ jobs:
 
   build-mac:
     name: Build Mac
-    runs-on: macos-latest
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        include:
+          - os: macos-latest
```

Comment:
maybe make this `macos-12` (the current equivalent), because at some point they'll move `latest` to mean `14` and then this will not work as intended.

Also maybe add `(x64)` or whatever in the description so its clear this is meant to be pinned to intel.

Reply:
Good points, thanks for the review! I Applied the suggested changes.